hikari/impl/rest.py

```diff
@@ -736,6 +736,7 @@ async def _request(
                     if not no_auth:
                         await live_attributes.still_alive().global_rate_limit.acquire()
 
+                    start = time.monotonic()
```

Comment:
This should only be registered when trace_logging is enabled, so this is incorrect

Reply:
This is for pyright support. We do much more for the sake of satisfying mypy.

Reply:
But this has runtime impact, which we dont want